src/test/java/codeu/controller/RegisterServletTest

```diff
@@ -0,0 +1,34 @@
+package codeu.controller;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class RegisterServletTest {
+
+ private RegisterServlet registerServlet;
+ private HttpServletRequest mockRequest;
+ private PrintWriter mockPrintWriter;
+ private HttpServletResponse mockResponse;
+
+ @Before
+ public void setup() throws IOException {
+   registerServlet = new RegisterServlet();
+   mockRequest = Mockito.mock(HttpServletRequest.class);
```

Comment:
Quick tip, you can actually annotate the variable with @Mock so you don't need to assign it here. That would mean deleting this line, and instead having "@Mock private HttpServletRequest mockRequest" on line 15 above.